Formula/t/tracker.rb

```diff
@@ -47,15 +47,13 @@ def install
       -Dsystemd_user_services=false
       -Dtests=false
       -Dsoup=soup3
+      --force-fallback-for=gvdb
```

Comment:
this would be an issue.

Reply:
I think I ran into some discussion about it, but currently not being able to find my PR

Reply:
Didn't think it'd be an issue since [criterion](https://github.com/Homebrew/homebrew-core/blob/4acd08b3cb1548f7da33709e3c8d9e06e4d4ff13/Formula/c/criterion.rb#L31) and [gegl](https://github.com/Homebrew/homebrew-core/blob/27e7ebc954dcb6b334f00f7d72669b41a0f70351/Formula/g/gegl.rb#L60) already use the flag.

Reply:
fair enough.